sequence_processing_pipeline/Pipeline.py

```diff
@@ -238,12 +254,21 @@ def generate_sample_information_files(self):
         Generate sample-information files in self.output_path.
         :return: A list of paths to sample-information-files.
         """
-        samples = []
-        for sample in self.sample_sheet.samples:
-            if sample['Sample_ID'].startswith('BLANK'):
-                samples.append((sample['Sample_ID'], sample['Sample_Project']))
+        if self.mapping_file is not None:
+            # TODO: Note we may need to introduce a conversion to sample_id
```

Comment:
I left this comment for discussion with you and MacKenzie. The BLANKS in the new mapping file are of the form '11052.BLANK242.11E'. Assuming that's correct form for sample-sheets, we can't assume that BLANKs start with 'BLANK'. Also, these are sample-names instead of the sample-ids we use when creating SIFs for the metagenomic pipeline; I would think we'd need to at least convert the dots to underscores and possibly strip the Qiita project ID from the beginning of the string.

In Pycharm if you begin a line with '# TODO' the line will appear blue instead of gray; subsequent lines begin one space in to retain the blue color. I'm running nose tests through ACT, which runs the GitHub workflows script including flake8.

Reply:
AFAIK that prefix added by Qiita ... but the notebook might do it too ... suggest checking.

Reply:
Will do!

Reply:
It appears that the notebook doesn't prepend the Qiita ID to the output mapping file so it sounds like it's added by Qiita.

Reply:
If we encounter sample-names prepended with qiita_ids, it sounds like we should go ahead and remove them. In that case we can return to using startswith().

Reply:
Based on feedback from MacKenzie and Gail, we'll assume that sample-names in the mapping files won't be prepended with Qiita IDs and go from there.